diff --git a/node_modules/whatsapp-web.js/src/util/Puppeteer.js b/node_modules/whatsapp-web.js/src/util/Puppeteer.js
index 54df700..eba12b8 100644
--- a/node_modules/whatsapp-web.js/src/util/Puppeteer.js
+++ b/node_modules/whatsapp-web.js/src/util/Puppeteer.js
@@ -17,7 +17,13 @@ async function exposeFunctionIfAbsent(page, name, fn) {
     if (exist) {
         return;
     }
-    await page.exposeFunction(name, fn);
+    try {
+        await page.exposeFunction(name, fn);
+    } catch (err) {
+        if (!/already exists/.test(err.message)) {
+            throw err;
+        }
+    }
 }
 
 module.exports = { exposeFunctionIfAbsent };
